### PR TITLE
LPS-159216 Convert colors in ColorPickerField inherited value

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/convertRGBtoHex.js [hunk 1]
```diff
@@ -12,20 +12,34 @@
  * details.
  */
 
+const RGB_REGEXP = /^rgb\((?<red>\d{1,3}),\s*(?<green>\d{1,3}),\s*(?<blue>\d{1,3})\)/;
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/convertRGBtoHex.js:15]: 😍 I love that you used groups!

**Comment by veroglez** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:forward `if Math.random() > 0.5`

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**